src/puppetlabs/puppetdb/cli/import.clj

```diff
@@ -14,7 +14,7 @@
             [puppetlabs.puppetdb.schema :refer [defn-validated]]
             [puppetlabs.puppetdb.utils :as utils
              :refer [base-url-schema export-root-dir]]
-            [puppetlabs.kitchensink.core :refer [cli!]]
+            [puppetlabs.kitchensink.core :as kitchensink :refer [cli!]]
```

Comment:
I just chose this because this is the convention we use in most of the other files